microreactors/mrad/mesh/HPMR_OneSixth_Core_meshgenerator_tri.i

```diff
@@ -13,6 +13,70 @@
 []
 
 [Mesh]
+  [BatchMeshGeneratorAction]
+    [cd0]
```

Comment:
same here, a comment like "# We generate each control drum in 3 steps, this syntax allows us to input this in a very compact manner"